README.md

```diff
@@ -1,68 +1,132 @@
 
-## ⚙️ Installation and start
+## 🚀 Getting Started
 
-Installation process is easy and takes minutes. See the [installation guide](/get-started/installation/).
+### HTML `<script>` Tag
 
-## Typescript support
+* Connect `Blocks` directly from your document:
 
-We use [JSDoc type annotations](https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html) for TypeScript static analysis support during development.
+```html
+<script src="https://unpkg.com/@uploadcare/blocks@{{PACKAGE_VERSION}}/web/blocks-browser.min.js" type="module"></script>
+```
 
-Additionally, we provide type definitions ([\*.d.ts files](https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html)) for the TypeScript projects in our packages.
+* Start using `Blocks` in your application markup:
 
-Check the [JSDoc Reference](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) page in TypeScript official documentation.
+```html
+<lr-file-uploader-regular
+  css-src="https://unpkg.com/@uploadcare/blocks@{{PACKAGE_VERSION}}/web/file-uploader-regular.min.css"
+>
+</lr-file-uploader-regular>
+```
 
-## Supported browsers
+### Using CLI
 
-Blocks are supported and tested in all major modern desktop and mobile browsers:
+You’ll need [Node.js](https://nodejs.org/) installed on your computer.
 
-- Chrome
-- Firefox
-- Safari
-- Edge
-- Opera
+* Install Blocks package: `npm i --save-exact @uploadcare/blocks`
+* Use `connectBlocksFrom` function to connect blocks:
+```js
+import { connectBlocksFrom } from '@uploadcare/blocks/abstract/connectBlocksFrom.js';
 
-Internet Explorer is outdated and not supported [anymore](https://uploadcare.com/blog/uploadcare-stops-internet-explorer-support/).
+connectBlocksFrom('https://unpkg.com/@uploadcare/blocks@{{PACKAGE_VERSION}}/web/blocks-browser.min.js');
+```
+* Start using `Blocks` in your application markup:
+```html
+<lr-file-uploader-inline
+  css-src="https://unpkg.com/@uploadcare/blocks@{{PACKAGE_VERSION}}/web/file-uploader-inline.min.css"
+  class="my-config-class"
+></lr-file-uploader-inline>
+```
 
-## Frameworks support
+### Advanced mode
 
-Blocks are a framework-agnostic solution, so you can use it with any runtime you like. We have prepared _examples_ for the most popular frameworks.
+* Install Blocks package: `npm i @uploadcare/blocks`
+* Start using blocks and build your own solutions from the source code:
+```js
+import * as LR from '@uploadcare/blocks';
+
+LR.registerBlocks(LR);
+```
+
+### Using File Uploading Features
+
+By default, `Blocks` use `demopublickey`, which you can save for demo purposes. To replace it with your own public key, sign up to [Uploadcare](https://app.uploadcare.com/accounts/signup/) and get a Public API key in [Uploadcare project's dashboard](https://app.uploadcare.com/projects/-/api-keys/). 
```

Comment:
Found the information here: https://uploadcare.github.io/blocks/solutions/file-uploader/

Let's discuss if we really need this website or how we can change it—it's yet another source of UNtruth (sometimes), but it has beautiful live demos here.

Reply:
All those docs in the repo are irrelevant a bit. We were going to remove them, drop this website and completely move docs to the https://uplodcare.com/docs/

And live demos actually not as good as I wanted to :)

Reply:
When are we gonna remove that section? Why can't we do it right now? I just found this information by googling, and it could be really confusing for anyone else.